Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Repod/2.9.0.221 Mozilla/5.0 (Linux; Android 11; Pixel 3 Build/RQ3A.210905.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/94.0.4606.85 Mobile Safari/537.36'
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: Repod
+    version: "2.9.0.221"
+  device:
+    type: smartphone
+    brand: Google
+    model: 'Pixel 3'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'rhythmbox/3.4.6'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Rhythmbox
+    version: "3.4.6"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'SachNoi.app/1 CFNetwork/1197 Darwin/20.0.0'
+  os:
+    name: iOS
+    version: "14.0"
+    platform: ""
+  client:
+    type: mobile app
+    name: SachNoi
+    version: "1"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'SachNoiApp/2 CFNetwork/1128.0.1 Darwin/19.6.0'
+  os:
+    name: iOS
+    version: "13.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: SachNoi
+    version: "2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'sp-agent'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Samsung Podcasts
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'sp-agent ETSfy24nuLJtoscVI1I0Fw'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Samsung Podcasts
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'sp-agent vQAsjA'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Samsung Podcasts
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'ServeStream Dynamo/1.0'
+  os: [ ]
+  client:
+    type: mobile app
+    name: ServeStream
+    version: "1.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'ServeStream'
+  os: [ ]
+  client:
+    type: mobile app
+    name: ServeStream
+    version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Shadow/45 CFNetwork/1335.0.3.1 Darwin/21.6.0'
+  os:
+    name: iOS
+    version: "15.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: Shadow
+    version: "45"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Shadowrocket/1858 CFNetwork/1331.0.7 Darwin/21.4.0'
+  os:
+    name: iOS
+    version: "15.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: Shadow
```

Comment:
I believe this app is https://apps.apple.com/sg/app/shadowrocket/id932747118 so name should be Shadowrocket